clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/DcosConfigurationProperties.groovy

```diff
@@ -19,6 +19,8 @@ class DcosConfigurationProperties {
     String dcosUrl
     String user
     String password
+    String serviceId
```

Comment:
Should this just be folded into the user field?

Would make my validation logic easier.